array-methods/map.js

```diff
@@ -0,0 +1,6 @@
+const prices = [42.42, 10, 28.2234, 3.2, 5, 12];
+
+const priceArray = prices.map((x) =>
+  console.log(`price: ${x} , salePrice: ${x / 2}`)
+);
```

Comment:
Per step 3 of `.map`, you were supposed to return an object not a string! 

You get the gist of `.map`, so no worries for now, but a heads up to be more careful next time!